src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -49,26 +52,99 @@ export class PatientViewPageStore
     {
         labelMobxPromises(this);
 
-        autorun(
-            ()=> {
-                if (!this.visibleRows || this.visibleRows.length === 0) {
+        reaction(() => {
+                    if (this.allDiscreteCNADataRequested) {
+                        return {
+                            allDiscreteCNADataRequested: this.allDiscreteCNADataRequested,
+                            sampleIds: this.samples.result.map((s:Sample)=>s.sampleId),
+                            discreteCNACache: this.discreteCNACache
+                        };
+                    }
+            },
+            (data:any) => {

```

Comment:
Don't type data here. The typings let TypeScript know the type and when you remove `:any` you will see that you have errors.